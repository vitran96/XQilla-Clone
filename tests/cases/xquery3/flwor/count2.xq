for $p in (
<p name="courgettes" sales="40"/>,
<p name="tomatoes" sales="500"/>,
<p name="aubergine" sales="320"/>,
<p name="runner beans" sales="602"/>,
<p name="dwarf beans" sales="26"/>,
<p name="potatoes" sales="2002"/>
)
order by number($p/@sales) descending
count $rank
where $rank <= 5
return
   <product rank="{$rank}"> {$p/@name, $p/@sales} </product>
