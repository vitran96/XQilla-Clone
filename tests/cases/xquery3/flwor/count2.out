<product rank="1" name="potatoes" sales="2002"/><product rank="2" name="runner beans" sales="602"/><product rank="3" name="tomatoes" sales="500"/><product rank="4" name="aubergine" sales="320"/><product rank="5" name="courgettes" sales="40"/>